<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeniuPrincipal</class>
 <widget class="QDialog" name="MeniuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VOT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #009b77;
color: dark;</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:   #008062;</string>
   </property>
   <widget class="QPushButton" name="pb_profil">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>14</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_profil{
	background-image: url(:/imagini/Imagini/user.png);
}
#pb_profil:hover { 
	background-color: #005944;
	background-image: url(:/imagini/Imagini/user.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_login">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_login{
	background-color: gray;
}
#pb_login:hover { 
	background-color: #6c6c6c;
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cauta">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>12</y>
      <width>290</width>
      <height>35</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ababab;
border-radius: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>                                    Cauta sesiuni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cauta">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_cauta{
	background-color: gray;
}
#pb_cauta:hover { 
	background-color: #6c6c6c;
}
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_admin_mode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: darck;
border-radius: 10px;
color: red;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string> Admin mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>200</width>
     <height>1140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #181818;
color: white;</string>
   </property>
   <widget class="QPushButton" name="pb_vActive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_vActive{
	background-color: #282828;
}
#pb_vActive:hover { 
	background-color: #454545;
}
</string>
    </property>
    <property name="text">
     <string>Sesiuni Active</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #282828;
font: 700 12pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Sesiuni de vot</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_vInactive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_vInactive{
	background-color: #282828;
}
#pb_vInactive:hover { 
	background-color: #454545;
}
</string>
    </property>
    <property name="text">
     <string>Sesiuni Inactive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_add_Vot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_add_Vot{
}
#pb_add_Vot:hover { 
	background-color: #282828;
}
</string>
    </property>
    <property name="text">
     <string>Adauga Sesiune</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_vot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_descriere">
       <property name="placeholderText">
        <string>            descriere</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_nr_opt">
       <property name="placeholderText">
        <string>       numar optiuni</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_status">
       <property name="placeholderText">
        <string> status: Activa/Inactiva</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_nr_opt_disponibile">
       <property name="placeholderText">
        <string> nr. optiuni disponibile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_nr_opt">
       <property name="styleSheet">
        <string notr="true">#pb_nr_opt{
}
#pb_nr_opt:hover { 
	background-color: #282828;
}
</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_opt">
       <property name="placeholderText">
        <string>              optiune</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_opt">
       <property name="styleSheet">
        <string notr="true">#pb-opt{
}
#pb_opt:hover { 
	background-color: #282828;
}
</string>
       </property>
       <property name="text">
        <string>Send options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="placeholderText">
        <string>         introduceti ID</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_activ">
         <property name="styleSheet">
          <string notr="true">#pb_activ{
}
#pb_activ:hover { 
	background-color: #282828;
}
</string>
         </property>
         <property name="text">
          <string>Activ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_inactiv">
         <property name="styleSheet">
          <string notr="true">#pb_inactiv{
}
#pb_inactiv:hover { 
	background-color: #282828;
}
</string>
         </property>
         <property name="text">
          <string>Inactiv</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
	background-color: #282828;
}
#pushButton:hover { 
	background-color: #454545;
}
</string>
    </property>
    <property name="text">
     <string>Rezultate Sesiuni Incheiate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_genereaza">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>145</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>     sesiune inactiva cautata</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
